internal/linker/link_node_modules.ts

```diff
@@ -189,37 +207,49 @@ export function main(args: string[], runfiles: Runfiles) {
   }
 
   // Create the $pwd/node_modules directory that node will resolve from
-  symlink(rootDir, 'node_modules');
+  await symlink(rootDir, 'node_modules');
   process.chdir(rootDir);
 
   // Symlinks to packages need to reach back to the workspace/runfiles directory
   const workspaceRelative = path.relative('.', workspaceDir);
   const runfilesRelative = runfiles.dir ? path.relative('.', runfiles.dir) : undefined;
 
   // Now add symlinks to each of our first-party packages so they appear under the node_modules tree
-  for (const m of Object.keys(modules)) {
+  const links = []
+
+  const linkModule = async (name:string,modulePath:string)=>{
```

Comment:
it doesn't look like the formatter ran over this.
It's configured as a pre-commit hook, can you check whether it's running for you?
(IIRC we figured it wasn't worth a CI check for formatting since it's nicely built into the commit hook)